### PR TITLE
embed new faq (google doc) in faq page

### PR DESCRIPTION
**Pull request by alisman**
image

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to master.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

### CODE REVIEW

**Comment by alisman** [comment]
@jjgao how does this look (screenshot)

**Comment by jjgao** [comment]
@alisman why there are two scroll bars?

**Comment by alisman** [comment]
@jjgao one is iframe scrollbar and one is browser scrollbar.   we could get rid of iframe scrollbar if we set the height of the iframe to the height we know the faq to be, but if/when the faq changes, that will have to be adjusted.

**Comment by jjgao** [comment]
Here is the heroku instance: https://cbioportal-pr-4624.herokuapp.com/faq.jsp

**Comment by jjgao** [comment]
@alisman is it possible to set the i-frame height to the window height to get rid of the page scrollbar?

**Comment by alisman** [comment]
@jjgao it is, but if FAQ changes, we will have to adjust.  but i will go ahead and do it.

**Comment by jjgao** [comment]
@alisman I mean change the i-frame height so that we can get rid of the outer/right scrollbar? Would that also require adjustment when FAQ changes?

**Comment by alisman** [comment]
ok, i get it.  just pushed a change.  kind of hacky but ...

File: portal/src/main/webapp/faq.jsp [hunk 1]
```diff
@@ -40,45 +40,41 @@
...
-
-        <script type="text/javascript" src="js/lib/showdown.min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/lib/showdown-github.min.js?${appVersion}"></script>
-        <script type="text/javascript" src="js/src/url_based_content.js?${appVersion}"></script>
         <script>
         window.loadReactApp({ defaultRoute: 'blank' });
+        
+        var to;
+        
+        function setIframeHeight() {
```
Comment (alisman) [inline portal/src/main/webapp/faq.jsp]: necessary to make iframe always approx fill the page without a scrollbar on browser portal

File: portal/src/main/webapp/faq.jsp [hunk 2]
```diff
@@ -40,45 +40,41 @@
...
-
-    <!-- Initialization script -->
-<script>
-    $(document).ready( function() {
-        // retrieve link for FAQ and generate the page
-        var faqLink = '<%= GlobalProperties.getFaqHtml()%>';
-        var baseUrl = '<%= GlobalProperties.getBaseUrl()%>';
-        var markdownDocumentation = '<%= GlobalProperties.isMarkdownDocumentation()%>';
-        var generatePage = new GeneratePage(baseUrl, faqLink, markdownDocumentation, "#faqPage");
-        generatePage.init();
```
Comment (inodb) [inline portal/src/main/webapp/faq.jsp]: Not sure if we should still support including your own FAQ

File: portal/src/main/webapp/faq.jsp [hunk 3]
```diff
@@ -40,45 +40,41 @@
...
+        });
+        
         </script>
     </jsp:attribute>
 
     <jsp:attribute name="body_area">
         <div id="reactRoot" class="hidden"></div>
-
-        <div id="faqPage"></div>
+        <iframe id="faqIframe" style="width:100%;border:1px solid #ddd;" src="https://docs.google.com/document/d/e/2PACX-1vSWTtIJZF2tuBimihr8ke-d00DpKh7fydFIQb5xYpE_bMYM9hZyY9OP1Vz1Ts0ow7ob-3h2S19cuB5O/pub?embedded=true"></iframe>
```
Comment (inodb) [inline portal/src/main/webapp/faq.jsp]: maybe this just as a property?

**Comment by jjgao** [comment]
fyi -- we are discussing whether we should move the faq to a markdown.

**Comment by jjgao** [comment]
closing this since we are moving the google doc to markdown.

**Comment by jjgao** [comment]
https://github.com/cBioPortal/cbioportal/pull/4809

**Comment by jjgao** [comment]
Somehow the google doc FAQ appeared on the alpha portal:

http://www.cbioportal.org/alpha/s/faq

Let's revert it.

https://github.com/cBioPortal/cbioportal/issues/4814

**Pull request closed without merging**